app/lib/reducers/ui.js

```diff
@@ -111,7 +111,7 @@ const reducer = (state = initial, action) => {
         }
 
         if (null != config.colors) {
-          if (state.colors.toString() !== config.colors.toString()) {
+          if (JSON.stringify(state.colors) !== JSON.stringify(config.colors)) {
```

Comment:
My concern specifically was for the people who will have the outdated format in their configs, and will be compared to a different new initial state?